components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -194,17 +193,39 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
       // When in this state, we expect it to become running or stopping anytime soon.
       case "interrupted":
         phase = StartPhase.Running;
-        statusMessage = <p className="text-base text-gray-400">Checking On Workspace …</p>;
+        statusMessage = <p className="text-base text-gray-400">Checking workspace …</p>;
         break;
 
       // Stopping means that the workspace is currently shutting down. It could go to stopped every moment.
       case "stopping":
-        statusMessage = <p className="text-base text-gray-400">Stopping …</p>;
+        phase = StartPhase.Stopping;
+        statusMessage = <div>
```

Comment:
**issue**: Can we center align all the elements within this div and also keep a fix width for the workspace reference?

Reply:
Moved to https://github.com/gitpod-io/gitpod/issues/3507.